docs/reference/ccr/apis/auto-follow/get-auto-follow-pattern.asciidoc

```diff
@@ -96,6 +96,7 @@ The API returns the following result:
         [
           "leader_index*"
         ],
+        "leader_index_exclusion_patterns": [],
```

Comment:
I would prefer to leave out the empty list here. But it could make sense to update the example to have an actual exclusion.